packages/fe-mockserver-core/test/unit/middleware.test.ts

```diff
@@ -102,7 +102,7 @@ describe('V4 Requestor', function () {
                 "/RootElement(ID=1,IsActiveEntity=true)/sap.fe.core.ActionVisibility.baseFunction(data='I am data')"
             )
             .execute('GET');
-        expect(dataRes).toMatchInlineSnapshot(`"%27I%20am%20data%27"`);
+        expect(dataRes).toMatchInlineSnapshot(`"'I am data'"`);
```

Comment:
this is not quite right yet, it has the single quotes

Reply:
yes almost :)

Reply:
should be better now